maat-court-data-api/src/main/java/gov/uk/courtdata/link/processor/OffenceInfoProcessor.java

```diff
@@ -16,6 +17,7 @@
 
 @RequiredArgsConstructor
 @Component
+@Slf4j
```

Comment:
Maybe we don't need this annotation here.

Reply:
As discussed, This is private Cons, Dont want to over engineer it with combinations of  annotations.

Reply:
Done